tests/app.py

```
@@ -1,16 +1,22 @@
+from oxapy import Oxapy, Router, get, post
 
 
-async def main():
-    await Oxapy(("127.0.0.1", 5555)).attach(Router().route(hello)).async_mode().run()
+def main():
+    (
+        Oxapy(("0.0.0.0", 3000))
+        .attach(
+            Router()
+            .route(get("/", lambda _: ""))
+            .route(get("/user/{id:int}", lambda _, id: str(id)))
```

Ruff (0.16.2)

[error] Lambda argument `id` is shadowing a Python builtin

(A006)